tests/by-util/test_rm.rs

```diff
@@ -698,3 +698,65 @@ fn test_non_utf8() {
     ucmd.arg(file).succeeds();
     assert!(!at.file_exists(file));
 }
+
```

Comment:
Both tests fail on my machine because the error message is `rm: cannot remove 'a': Device or resource busy (os error 16)`